server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MessageContentExtractor.java

```diff
@@ -84,6 +104,11 @@ private MessageContent parseMultipartAlternative(Multipart multipart) throws IOE
         return new MessageContent(textBody, htmlBody);
     }
 
+    private MessageContent parseMultipartRelated(Multipart multipart) throws IOException {
+        Optional<String> htmlBody = getFirstMatchingTextBody(multipart, "text/html");
```

Comment:
Seeing https://tools.ietf.org/html/rfc2387#section-5.2 it is possible.

We shoul, if there is no HTML, try to find related text... (Maybe it is not even text/plain as the example I gave suggest)